include/swift/SIL/MemAccessUtils.h

```diff
@@ -53,6 +53,14 @@ namespace swift {
 /// (pointer-to-address is not stripped).
 SILValue stripAccessMarkers(SILValue v);
 
+/// Return a non-null address-type SingleValueInstruction if \p v is the result
+/// of an address projection that may be inside of a formal access, such as
+/// (begin_borrow, struct_element_addr, tuple_element_addr).
+///
+/// The resulting projection must have an address-type operand in position zero
```

Comment:
What is position zero here? Can you explain that in more standard language? (Just putting on my undefined variable hat).

Reply:
How can I possibly be more specific about this? What word is "non-standard" here?

Reply:
Just weird to hear "position zero" instead of zeroth operand. My suggestion:

The resulting projection must have an address-typed operand zero? That being said. I don't care that much about this. I see your point. LGTM.